packaging/create_rpm

```diff
@@ -248,7 +255,11 @@ def download_listed_resources(resource_file, destination_path):
         if (not resource) or resource.startswith('#'):
             # Cope with blank lines and comments
             continue
-        resource_file_name = os.path.split(resource)[-1]
+        resource_file_name = os.path.basename(resource)
+        resource_file_prefix = RPM_PREFIX.findall(resource_file_name)[0]
```

Comment:
A little bit more comments here would also go a long way (e.g. how we are determining when to use local packages and when not to)